runtime/compiler/build/toolcfg/common.mk

```diff
@@ -60,10 +60,6 @@ ifdef ENABLE_GPU
         $(error You must set CUDA_HOME if ENABLE_GPU is set)
     endif
 
-    ifeq (,$(GDK_HOME))
-        $(error You must set GDK_HOME if ENABLE_GPU is set)
-    endif
-
     PRODUCT_INCLUDES+=$(CUDA_HOME)/include $(CUDA_HOME)/nvvm/include $(GDK_HOME)
```

Comment:
I guess the `$(GDK_HOME)` in this line is no longer needed.

Reply:
It's not a problem if `GDK_HOME` is empty.

Reply:
Yes, it's still needed for versions 7.5 or older, in which case you'll get a compile error.